data/remote_control_interface_web/js/script.js

```
@@ -0,0 +1,275 @@
+    const deadzoneBindings = [
+        ["deadzone-left-x", "dead_zone_left", "x"],
+        ["deadzone-left-y", "dead_zone_left", "y"],
+        ["deadzone-right-x", "dead_zone_right", "x"],
+        ["deadzone-right-y", "dead_zone_right", "y"]
+    ];
+
+    deadzoneBindings.forEach(([id, group, axis]) => {
+        const slider = document.getElementById(id);
+        slider.value = joystickState[group][axis];
+        slider.oninput = () => {
+            joystickState[group][axis] = parseFloat(slider.value);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)